packages/core/src/types.ts

```diff
@@ -2186,6 +2186,8 @@ export interface ActorRef<
 
 export type AnyActorRef = ActorRef<any, any, any>;
 
+export type SomeActorRef = ActorRef<Snapshot<unknown>, EventObject>;
```

Comment:
I'd call this `UnknownActorRef`. It might fit better some other existing types of this kind

Reply:
works for me